### DISCUSSION TITLE
Could not load file or assembly CefSharp.Core.Runtime

### DISCUSSION THREAD

**Discussion by sinanakyazici** [cefsharp/CefSharp#3646] [General]
![image](https://user-images.githubusercontent.com/1764402/123435871-b6b58880-d5d6-11eb-9368-20e7afc662c8.png)

I created a .netcore 5.0 application. I installed offscreen.core. But the application doesn't start. You can see the error on image.
I installed vc++ 2013,2015-2019, what can I do else ?

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/issues/3284#issuecomment-772132523

https://github.com/cefsharp/CefSharp.MinimalExample includes working example.
